apollo-api/src/commonMain/kotlin/com/apollographql/apollo3/api/test/MapBuilder.kt

```diff
@@ -63,7 +67,7 @@ class MandatoryTypenameProperty {
       property: KProperty<*>,
   ): String {
     check(typename != null) {
-      "__typename is not known at compile-time for this type. Please specify it explicitely"
+      "__typename is not known at compile-time for this type. Please specify it explicitly"
```

Comment:
Unrelated but noticed this typo 😅  Was also elsewhere.

Reply:
Yep, that's me! Thanks!